python/cugraph/cugraph/utilities/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources path_retrieval_wrapper.pyx)
+set(linked_libraries cugraph::cugraph)
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX utilities_
+)
+
+set(targets_using_numpy utilities_path_retrieval_wrapper)
```

Comment:
@vyasr Can these foreach loops be merged together?

```cmake
foreach(cython_module IN LISTS RAPIDS_CYTHON_CREATED_TARGETS)
  target_include_directories(${cython_module} PRIVATE "${Python_NumPy_INCLUDE_DIRS}")
  set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH "\$ORIGIN;\$ORIGIN/../library")
endforeach()
```

Reply:
In this case yes since there's only one module. @jnke2016 does this actually require the numpy headers, though, or was leaving it here just an oversight?